src/index.js

```diff
@@ -174,19 +197,14 @@ export function createActionLogger(options: ActionLoggerOptionsType): Function {
       }
     }
 
-
-    if (endpoint.transformFunction) {
```

Comment:
Moving this makes sense, the transform may not be necessary if there's a custom endpoint function